### ISSUE TITLE
AWS v4 Signatures

### ISSUE THREAD

**Issue by joeauty** [adamhenson/s3-image-uploader#3]
Hello,

Just wondering if you plan to support (I'm presuming by updating a dependency) AWS v4 signatures? I'm getting the following error uploading to my CA-Central bucket:

`The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256.`

It works with with US-East.

**Comment by adamhenson** [comment]
No, sorry @joeauty. To be honest - I don't know what that is, and I don't think I will be adding many more features to this package.

Are you adding the `key` and `secret`? Mind sharing some code?
